adding to modules map: Test/hsa1.c -> foo
setting default module to default
note: the -fo-merge-E_variable-and-var-values flag is incompatible with doing module analyses, so turning it off.

---- START ---- Test/hsa1.c
--- Test/hsa1.c
+++ Test/hsa1.c
@@ -7,1 +7,1 @@
-  return i;
+  {return i;}
@@ -16,1 +16,1 @@
-  int b[10];
+  int (*b)[10]=(typeof(b))xmalloc(sizeof *b);
@@ -20,1 +20,1 @@
-    b[i] = 10 * i;
+    (*b)[i] = 10 * i;
@@ -23,1 +23,1 @@
-  f(b[3], 3, &b[2]);
+  f((*b)[3], 3, &(*b)[2]);
@@ -25,1 +25,1 @@
-  return 0;
+  {free(b);return 0;}
---- STOP ----

